geode-core/src/integrationTest/java/org/apache/geode/internal/cache/PartitionedRegionSingleNodeOperationsJUnitTest.java

```diff
@@ -108,7 +108,8 @@ public void test000Put() throws Exception {
     logger.info("<ExpectedException action=add>" + expectedExceptions + "</ExpectedException>");
     try {
       pr.put(new Integer(1), val);
-      fail("testPut()- Expected PartitionedRegionException not thrown for localMaxMemory = 0");
+      fail(
```

Comment:
If we're updating this file it might be nice to update the name `test000put` but I wouldn't block this PR on something minor

Reply:
Unfortunately I don't know that we can really do better with the naming without refactoring (or more likely rewriting) this whole class. The methods are numbered because they have to be run in order or they break, and this test does waaaayyy too much in one method to give it a proper name. I would like to rename it though